docs/topics/gradle/get-started-with-jvm-gradle-project.md

```diff
@@ -72,7 +72,7 @@ application {
 
 * 1️⃣ [Application plugin](https://docs.gradle.org/current/userguide/application_plugin.html) to add support for building CLI application in Java.
 * 2️⃣ Lean more about [sources of dependencies](https://docs.gradle.org/current/userguide/declaring_repositories.html).
-* 3️⃣ The [Maven Central Repository](https://search.maven.org/). It can also be [Google's Maven repository](https://maven.google.com/web/index.html) or your company's private repository.
+* 3️⃣ The [Maven Central Repository](https://central.sonatype.com/). It can also be [Google's Maven repository](https://maven.google.com/web/index.html) or your company's private repository.
```

Comment:
I think Goole repository is better to change to `https://maven.google.com`. Google automatically redirects to current URL. In the future, if redirection will be changed, we don't need to update our documentation.